mesop/component_helpers/style.py

```diff
@@ -73,12 +73,27 @@ class BorderSide:
                                 a string with a unit (e.g., '2em'), or None for no width.
       color: The color of the border, represented as a string. This can be any valid CSS color value,
                           or None for no color.
-      style: The style of the border, which can be 'none' for no border, 'solid' for a solid line.
+      style: The style of the border.
+      https://developer.mozilla.org/en-US/docs/Web/CSS/border-style
```

Comment:
[nit] I think this should go on the same line as the style arg.

```
The style of the border. See https://developer.mozilla.org/en-US/docs/Web/CSS/border-style
```

Reply:
Sure will do the change